portals/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx

```diff
@@ -761,7 +761,7 @@ class APIDefinition extends React.Component {
                         {api.type !== API.CONSTS.APIProduct && (
                             <ImportDefinition setSchemaDefinition={this.setSchemaDefinition} />
                         )}
-                        {api.serviceInfo && api.serviceInfo.outdated && (
+                        {api.serviceInfo && api.serviceInfo.outdated && api.type !== 'SOAP' && (
```

Comment:
Better to add extra brackets like follows to make the code more readable
```
(api.serviceInfo && api.serviceInfo.outdated && api.type !== 'SOAP')
```

Reply:
Fixed